### DISCUSSION TITLE
CSS styling question on aligning items

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#990] [Q&A] [answered]
### Question

Hi All,

This is more of a CSS question, which I am really unfamiliar with. 

So what I want to do is to have a left drawer and a header text inside my ```ui.header()```
If I use the CSS class ```justify-between``` I realized they will be aligned to the complete left and right.

However, the header text, I would like to have center-aligned in middle of my page.
I managed to get this done using another dummy label without any text like this below:
``` python
with ui.header(elevated=True).style('background-color: #3874c8').classes('items-center justify-between'):
    ui.button().props('flat color=white icon=menu')
    ui.label('This is my test header')
    ui.label()
```
![image](https://github.com/zauberzeug/nicegui/assets/65127974/8832f3ab-02ab-4a5a-a538-ffdfc0c27adc)

But is there a proper way to do this using some CSS method? Basically, if I have 2 items inside my header, can I have one on left, and the next one in center?

Any advice is much appreciated.

Thanks a lot,
Anindya

**Comment by falkoschindler** [comment, marked as answer]
Hi @Anindya088,

There are handy [Quasar classes for positioning elements](https://quasar.dev/style/positioning). For centering a label, you can use "absolute-center" like this:

```py
with ui.header():
    ui.button().props('flat color=white icon=menu')
    ui.label('This is my test header').classes('absolute-center')
```

**Reply by Anindya088** [reply]
Thanks a lot @falkoschindler. This is great. I was searching directly on the Tailwind website, did not know that we can get CSS classes from Quasar website also. Thanks so much for sharing this, very helpful.

Regards,
Anindya
